dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
```

Comment:
Is this member initialized to something by default, or set to null until the app sets a configuration or gets the system default?

Reply:
At WinAppSDK layer we would be setting it as null or empty string as default, on the OS Shell layer this empty value would be mapped to System default. Device Enumeration would happen at Shell OS layer.

Reply:
This value will be initialized to nullptr by default.